### PR TITLE
Test Multivector storage mmap and uint8

### PR DESCRIPTION
**Pull request by agourlay**
Follow up on https://github.com/qdrant/qdrant/pull/4194

### CODE REVIEW

File: tests/openapi/openapi_integration/test_multi_vector_uint8.py [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+        path_params={'collection_name': collection_name},
+        body={"limit": 10, "with_vector": True}
+    )
+    assert response.ok
+    assert len(response.json()['result']['points']) == 3
+    results = response.json()['result']['points']
+
+    first_point = results[0]
+    assert first_point['id'] == 1
+    assert first_point['vector']['my-multivec'] == [[1, 1, 1, 1], [2, 2, 2, 2], [3, 3, 3, 3]]
```
Comment (IvanPleshkov) [inline tests/openapi/openapi_integration/test_multi_vector_uint8.py:109]: I like your check with `f32 -> uint8` conversion

File: tests/openapi/openapi_integration/test_multi_vector_uint8.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+        method="POST",
+        path_params={'collection_name': collection_name},
+        body={
+            "points": [2]
+        }
+    )
+    assert response.ok
+
+    # retrieve by id after deletion
+    response = request_with_validation(
```
Comment (IvanPleshkov) [inline tests/openapi/openapi_integration/test_multi_vector_uint8.py:135]: Do we actually need this check here?

**Review by IvanPleshkov — APPROVED** [review]
Perfect tests, thanks!

File: tests/openapi/openapi_integration/test_multi_vector_uint8.py (hunk 2 above)
Comment (agourlay) [reply tests/openapi/openapi_integration/test_multi_vector_uint8.py:135]: not mandatory but it cannot hurt :)

**Pull request merged**